test/types/request.test-d.ts

```diff
@@ -86,7 +87,7 @@ const getHandler: RouteHandler = function (request, _reply) {
   expectType<string>(request.id)
   expectType<FastifyLoggerInstance>(request.log)
   expectType<RawRequestDefaultExpression['socket']>(request.socket)
-  expectType<Error & { validation: any; validationContext: string } | undefined>(request.validationError)
+  expectType<Error & { validation?: FastifySchemaValidationError[]; validationContext: string } | undefined>(request.validationError)
```

Comment:
It is correct the 99% of the times, but if a user customize the schema controller (eg: https://github.com/Eomm/joi-compiler/?tab=readme-ov-file#usage ) the `validation` does not contain a `FastifySchemaValidationError` anymore

Reply:
Hmmm but the code as linked above suggests, `validation` only gets added if the code hits the `const error = schemaErrorFormatter(result, dataVar)` section? Which already only accepts an input typed as `FastifySchemaValidationError[]`? 

Correct me if I have misinterpreted that 🙂